astropy/table/pprint.py

```diff
@@ -546,7 +546,10 @@ def format_col_str(idx):
                 except ValueError:
                     raise ValueError(
                         f'Unable to parse format string "{col_format}" for '
-                        f'entry "{col[idx]}" in column "{col.info.name}"'
+                        f'entry "{col[idx]}" in column "{col.info.name}" '
+                        f'with datatype "{col.info.dtype}". '
```

Comment:
a newline here reduce the chance of the url ending up truncated in the terminal, which is helpful for anyone relying on manual copy-pasting to explore it
```suggestion
                        f'with datatype "{col.info.dtype}".\n'
```